catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/ChartRepository.java

```diff
@@ -120,7 +121,8 @@ public void restorePatchAttributes(Chart original, Chart updated) {
 
   private EntityReference getService(Chart chart) throws IOException {
     EntityReference ref =
-        EntityUtil.getService(daoCollection.relationshipDAO(), Entity.CHART, chart.getId(), Entity.DASHBOARD_SERVICE);
+        EntityUtil.getService(
+            daoCollection.relationshipDAO(), Entity.CHART, chart.getId(), Entity.DASHBOARD_SERVICE, Include.ALL);
```

Comment:
Instead of passing Include.ALL, we can pass the flag based on the deleted state of entity `chart`, right?

Reply:
@harshach thinks so too. I'll change it, it makes sense to me too.

Reply:
Done